share/docker/Dockerfile

```diff
@@ -193,17 +195,9 @@ RUN gem install bundler -v 2.4.22 --no-document && \
     cd /tmp/lang/ruby && bundle install
 
 # Install Rust
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.73.0
-
-# Note: This "ubertool" container has two JDK versions:
-# - OpenJDK 8
-# - OpenJDK 11
-# - OpenJDK 17
-# - The root build.sh script switches between the versions according to
-#   the JAVA environment variable.
-
-# Since we want the JDK8 as a default, we have to re-prepend it to the PATH.
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
-ENV PATH $JAVA_HOME/bin:$PATH
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.74.0
```

Comment:
1.73.0 is MSRV in `main`. I believe that it should work in `branch-1.11` too. But it does not matter much anyway.